type.proto

```diff
@@ -73,6 +73,32 @@ enum ResponseStatus{
     active_disruption = 10;
 }
 
+enum FreeFloatingType {
+    BIKE = 1;
+    SCOOTER = 2;
+    MOTORSCOOTER = 3;
+    STATION = 4;
+    CAR = 5;
+    OTHER = 6;
```

Comment:
We've always set enums' fields as lower case. 
Please, follow the same conventions as the one already defined in the current file. This is clean-code #1 rule ! 
https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#general-rules